### ISSUE TITLE
Could we get a helper to convert an Entry to an Array?

### ISSUE THREAD

**Issue by stuartcusackie** [edalzell/statamic-blade#73]
I'm just looking for an easy way to convert an Entry object to the same format provided by your AugmentationViewServiceProvider. Could we get a helper function added for this? I have a temporary solution below that might explain what I'm trying to do.

This is the class I have written. It's basically the same code from your `AugmentedView `class. Maybe we can add this to the package's  'Augmentation' folder?

```
<?php

namespace App;

use Edalzell\Blade\Concerns\AugmentsValues;

class AugmentedEntry
{
    use AugmentsValues;

    public function gatherEntryData($entry): array
    {
        return array_map(function ($data) {
            return $this->getAugmentedValue($data);
        }, $entry->toAugmentedArray());
    }
}
```

And here is my helper function:

```
if(!function_exists('augment_entry')) {

	/**
	 * Use AugmentedEntry class to convert
	 * entry to array. Uses a trait from Edalzell's
	 * blade directives package.
	 * 
	 * @param $entry \Statamic\Entries\Entry
	 * @return array
	 */
	function augment_entry(\Statamic\Entries\Entry $entry): array
	{
		$augmentor = new \App\AugmentedEntry();
		return $augmentor->gatherEntryData($entry);
	}
}
```

I can submit a PR if you think it's useful. Thanks!

**Comment by edalzell** [comment]
Sorry for the silence @stuartcusackie been super busy. I'll try to take a look at all the issues/comments this week.

**Comment by stuartcusackie** [comment]
> Sorry for the silence @stuartcusackie been super busy. I'll try to take a look at all the issues/comments this week.

No problem at all! I'm getting on fine with my Statamic Blade builds and there's no major issues.

**Comment by edalzell** [comment]
Ya PR would be good.

**Comment by stuartcusackie** [comment]
Done!

**Comment by edalzell** [comment]
Released in 3.6

**Comment by edalzell** [comment]
released in 3.7
